hammers/scripts/conflict_macs.py

```diff
@@ -77,20 +77,36 @@ def main(argv=None):
         return -1
 
     # Do actual work
-    with slack or nullcontext():
-        conflict_macs = find_conflicts(auth, ignore_subnets, slack=slack)
+    try:
+        conflict_macs = find_conflicts(auth, ignore_subnets)
 
         if args.mode == 'info':
-            show_info(conflict_macs, slack=slack)
+            show_info(conflict_macs)
         elif args.mode == 'delete':
-            delete(auth, conflict_macs,
-                safe=not args.force_sane, slack=slack, verbose=args.verbose)
+            if (not args.force_sane) and len(conflict_macs) > 10:
+                raise RuntimeError('(in)sanity check: thinks there are {} conflicting MACs'.format(len(conflict_macs)))
+            
+            for mac in conflict_macs.values():
+                osrest.neutron_port_delete(auth, mac['neutron_port_id'])
+
+            if slack:
+                message = 'Fixed Ironic/Neutron MAC conflicts\n{}'.format(
```

Comment:
Good rule for logs: log things that happened, not things that are supposedly going to happen. Most logs were like "About to do this..." when it's much more helpful to say "Did this." or "Tried to do this, but failed." Logs can lie, especially when they are talking about hypothetical things. A log could indicate that X is going to happen, but then an error could occur preventing X from even starting.